samples/coroutines-loader/src/main/kotlin/money/vivid/elmslie/samples/coroutines/timer/elm/TimerActor.kt

```diff
@@ -16,7 +16,7 @@ internal object TimerActor : Actor<Command, Event>() {
                     errorMapper = { Event.OnTimeError(it) },
                 )
 
-            is Command.Stop -> cancelSwitchFlow<Command.Start>()
+            is Command.Stop -> cancelSwitchFlowsAsFlow(Command.Start::class.simpleName.orEmpty())
```

Comment:
I think, will be better to use only custom keys, if we remove generic from cancelSwitchFlow, since this expression looks complex and agly a bit(

Reply:
left only a key version of switch function, with defaults of actor class name